go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/bufbuild/buf
 
-go 1.20
+go 1.21
```

Comment:
It seems like adding the `github.com/quic-go/quic-go` dependency bumped this up. I'm unsure why. If it's a problem I can look into this.

Reply:
It's a problem for a few more weeks - we need to keep compat back for three versions for our enterprise customers who install from source. Can either try to fix now, or just hold off trying to merge this until 1.23 is released - up to you.

Reply:
I think we can just wait. There's absolutely no rush for me.